### ISSUE TITLE
Unable to use warp-deploy with Node v8.x

### ISSUE THREAD

**Issue by ghoullier** [WarpJS/beta#12]
Node Version: v8.17.0

Command
```
> warp-deploy --asset-dir=dist .
```

Output
```
> warpjs-webpack-template@1.0.0 deploy /Users/ghoullier/Dev/ghoullier/warp-experiments
> warp-deploy --asset-dir=dist .

(node:59142) UnhandledPromiseRejectionWarning: ReferenceError: URL is not defined
    at Promise (/Users/ghoullier/Dev/ghoullier/warp-experiments/node_modules/@warpjs/webpack-plugin/lib/admin/deployServerArchive.js:1:8440)
    at new Promise (<anonymous>)
    at /Users/ghoullier/Dev/ghoullier/warp-experiments/node_modules/@warpjs/webpack-plugin/lib/admin/deployServerArchive.js:1:8271
    at t.readdir (/Users/ghoullier/Dev/ghoullier/warp-experiments/node_modules/@warpjs/webpack-plugin/lib/admin/deployServerArchive.js:1:9414)
    at FSReqWrap.oncomplete (fs.js:135:15)
(node:59142) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 4)
(node:59142) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by jpl-scale** [comment]
Thanks you for your report.

**Comment by jpl-scale** [comment]
Fixed in @warpjs/engine@1.1.1
